src/views/states/list/components/StatusBox.tsx

```diff
@@ -11,10 +10,9 @@ type StatusBoxProps = PropsWithChildren<{
   loaded: boolean;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   error: any;
```

Comment:
Cannot we here use some `Error` type? Or does it have to be of type `any`?

Reply:
No. `useK8sWatchResource` use any in error as it can contain also href for reference that is not defined in the Error type